addons/mail/static/src/models/emoji.js

```diff
@@ -0,0 +1,30 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+import { insertAndReplace } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'Emoji',
+    identifyingFields: ['unicode'],
+    fields: {
+        unicode: attr({
+            readonly: true,
+            required: true,
+        }),
+        sources: attr({
+            readonly: true,
+            required: true,
+        }),
+        description: attr({
+            readonly: true,
+            required: true,
+        }),
+        emojiRegistry: one('EmojiRegistry', {
+            default: insertAndReplace(),
+            inverse: 'allEmojis',
+            readonly: true,
+            required: true,
+        })
```

Comment:
Missing trailing comma.

We like trailing commas because they make it easier to reorder items and avoid creating meaningless diff on adjacent lines when adding new ones.